source/includes/_websocket-cash.md

```diff
@@ -1,275 +1,363 @@
+  "errorCode":   "NULL_VAL",
+  "cat":         "CASH",
+  "ei":          "NULL_VAL"
+}
+```
+
+Once connected to websocket streams, you will start receiving real time update of your own orders. It contains both order execution report and current balances. 
+Since only new order updates will be streamed, it is recommendated that you load the initial snap of all you orders using the [RESTful API GET api/v1/order/open](https://github.com/bitmax-exchange/api-doc/blob/master/bitmax-api-doc-v1.2.md#list-of-all-open-orders).
+
+
+
+
+
+Field          | Data Type  | Description
+-------------- | ---------- | ---------------------------------------
+  `m`          | String     | `"order"`
+  `execId`     | String     | for each user, this is a strictly increasing long integer
+  `coid`       | String     | client order id
+  `origCoid`   | String     | optional, the original order id, see canceling orders for more details 
+  `orderType`  | String     | `"Limit", "Market", "StopLimit", "StopMarket"`
+  `s`          | String     | symbol, e.g. `"BTC/USDT"`
+  `t`          | Long       | UTC timestamp in milliseconds
+  `p`          | String     | optional, limit price, only available for `"Limit"` and `"StopLimit"` orders
+  `sp`         | String     | optional, stop price, only available for stop market and stop limit orders
+  `q`          | String     | order quantity
+  `l`          | String     | last quantity, the quantity executed by the last fill 
+  `f`          | String     | filled quantity, this is the aggregated quantity executed by all past fills
+  `ap`         | String     | average filled price
+  `bb`         | String     | base asset total balance
+  `bpb`        | String     | base asset available balance
+  `qb`         | String     | quote asset total balance
+  `qpb`        | String     | quote asset available balance
+  `fee`        | String     | fee
+  `fa`         | String     | fee asset 
+  `bc`         | String     | if possitive, this is the BTMX commission charged by reverse mining, if negative, this is the mining output of the current fill.
+  `btmxBal`    | String     | optional, the BTMX balance of the current account.
+  `side`       | String     | side
+  `status`     | String     | order status
+  `errorCode`  | String     | if the order is rejected, this field explains why
+  `cat`        | String     | category
+  `ei`         | String     | execution instruction, `"POST"` for post-only orders
+
+### Data Channel - Balance Updates
+
+> Balance Update Data
+
+```json
+{
+  "m":  "balance",
+  "execId":  "31428578",
+  "account": "cshnHfBk4eytjL23VBZ0mRTY2Dre6Tcr",
+  "a": "USDT",
+  "b":  "177718.014527793",
+  "pb":  "166707.014527793" 
+}
+```
+Once connected to websocket streams, you will start receiving real time update of your balances. Any change of your balances which is not due to an order will be streamed to you. (The balance change due to an order is streamed as an order update message)
```

Comment:
real time balance updates.  Trade related balance updates will be included in the order message, all other updates (deposit/withdraw/transfer between cash and margin accounts, etc) will be included in the balance message.